.circleci/config.yml

```diff
@@ -87,8 +87,8 @@ commands:
           name:  "Install Rust"
           command: |
             curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain $(< rust-toolchain) && source $HOME/.cargo/env
-            rustup install nightly-2019-07-31
-            rustup component add rustfmt --toolchain nightly-2019-07-31
+            rustup install nightly-2020-01-15
```

Comment:
That is already merged by now, feel free to rebase :)

Reply:
Cool, cheers.